doc/whats_new.rst

```diff
@@ -145,6 +145,8 @@ Bug
 
 - Fix problem with :meth:`mne.io.Raw.add_channels` where ``raw.info['bads']`` was replicated by `Eric Larson`_
 
+- Fix bug when trying to filter epochs loaded from disk with :func:`mne.read_epochs` by `Eric Larson`_
```

Comment:
this is not a bug that was released before no? if it's a bug fix from a PR we just merged we should not put this in what's new

Reply:
Actually the bug is with `endianness` introduced by that PR, I'll fix that, too, and remove `whats_new.rst` update